docs/content/en/docs/crd-ref/metrics/v1alpha3/_index.md

```diff
@@ -138,6 +189,70 @@ _Appears in:_
 | `secretKeyRef` _[SecretKeySelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#secretkeyselector-v1-core)_ | SecretKeyRef defines an optional secret for access credentials to the metrics provider. |
 
 
+#### ObjectReference
+
+
+
+
+
+_Appears in:_
+- [Objective](#objective)
+
+| Field | Description |
+| --- | --- |
+| `name` _string_ |  |
+| `namespace` _string_ |  |
+
+
+#### Objective
+
+
+
+Objective defines an objective for analysis
+
+_Appears in:_
+- [AnalysisDefinitionSpec](#analysisdefinitionspec)
+
+| Field | Description |
+| --- | --- |
+| `analysisValueTemplateRef` _[ObjectReference](#objectreference)_ | AnalysisValueTemplateRef defines a reference to the used AnalysisValueTemplate |
+| `target` _[Target](#target)_ | Target defines failure or warning criteria |
+| `weight` _integer_ | Weight can be used to emphasize the importance of one Objective over the others |
+| `keyObjective` _boolean_ | KeyObjective defines if the objective fails when the target is not met |
```

Comment:
```suggestion
| `keyObjective` _boolean_ | If KeyObjective is true, the objective fails when the target is not met |
```

Reply:
We need to start the comment with the name of the parameter we are describing, this is required in technical documentations